make.sh

```diff
@@ -189,12 +189,6 @@ rm -rf .github/workflows/test_install_script.yml
 rm -f .git/index
 git reset
 
-if [[ -z "${CI}" ]]; then
```

Comment:
Why you removed this?

Reply:
Cause we don't use deliverable_setup.sh anymore, so I removed this. I will restore the rm part

Reply:
I think we still need to use it, but with Swift Fastlane, please revert the changes

Reply:
Done

Reply:
@ducbm051291 Please revert all removed changes:
- make.sh
- set_up_test_testflight.sh
- deliverable_setup.sh

If the workflow does not work, please create a new task for that

Reply:
Got it ;)